README.md

```diff
@@ -0,0 +1,38 @@
+Instructions to build Python environment
+
+Linux, using Python 2.7, system packages
+
+TBD
```

Comment:
python-pip I think might also be important.. I don't use ubuntu so it might be difficult for me to come up with an ubuntu specific package installation but I can look into it